src/config/settings.py

```diff
@@ -0,0 +1,12 @@
+from pydantic.v1 import BaseSettings
+
+
+class Settings(BaseSettings):
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature.db"
+
+    class Config:
+        case_sensitive = True
+        enf_file = "../../.env"
```

Comment:
There is a typo in the `Config` class. It should be `env_file` instead of `enf_file` to correctly specify the path to the environment file.